Cargo.toml

```diff
@@ -14,4 +14,16 @@ members = [
     "sgxs-loaders",
     "sgxs-tools"
 ]
+default-members = [
```

Comment:
Now that not everything is built by default, CI should be modified so the excluded packages are still tested. Or, you can just not set this.